cypress/e2e/canonicity/tables.cy.js

```diff
@@ -14,6 +14,18 @@ suite(["@CI"], "table", () => {
       url: `/addresses/accounts/${DEFAULT_ACCOUNT_PK}`,
       section: "SNARK Jobs",
     },
+    {
+      url: "/summary",
+      section: "Blocks",
+    },
+    {
+      url: "/",
+      section: "Blocks",
+    },
```

Comment:
I feel like these snuck back in after https://github.com/Granola-Team/mina-block-explorer/commit/b21f56f69449665f592547df48bde863cb118109. Are they still necessary?